docs/3.0.0-beta.x/getting-started/quick-start-tutorial.md

```diff
@@ -622,7 +623,7 @@ Forbidden Access Looks like this:
 
 ::: tip NOTE
 
-If you would like to see the route of any specific **Content Type**, you need to navigate to the **Content Type** under the **Roles and Permissions** plugin and click the ⚙️ next to the value. On the right, you see the route:
+If you would like to see the route of any specific **Collection Type**, you need to navigate to the **Collection Type** under the **Roles and Permissions** plugin and click the ⚙️ next to the value. On the right, you see the route:
```

Comment:
You should use `Content Type` in this sentence, this is a tip about a global concept that concern Collection and Single Types.

Reply:
All requested changes are very obvious to the point that I do not understand how I missed them.

Reply:
No problem, thank for the update. I really appreciate.